pallets/automation-time/src/lib.rs

```diff
@@ -349,63 +349,21 @@ pub mod pallet {
 
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
-		fn on_initialize(_: T::BlockNumber) -> Weight {
+		fn on_initialize(block: T::BlockNumber) -> Weight {
```

Comment:
If the block variable is not used in this function, maybe we should call it `_block`? ( `_` is definitely not meaningful and should not be used.)

From ChatGPT:
In Rust, using an underscore (`_`) as a variable or parameter name is a convention to indicate that the value is intentionally unused or ignored. It is used when a variable or parameter is required by the language syntax but not actually needed within the code block.

In the specific example you provided (`fn on_initialize(block: T::BlockNumber) -> Weight`), if the `block` parameter is not used within the body of the `on_initialize` function, an underscore (`_`) could be used instead to indicate that the parameter is intentionally ignored. This can be helpful for code readability and to avoid compiler warnings about unused variables.

Here's an example of how the code could be written using `_` to indicate the parameter is unused:

```rust
fn on_initialize(_block: T::BlockNumber) -> Weight {
    // Code logic here
    // The `_block` parameter is intentionally ignored
}
```

Using `_` as a variable or parameter name is not specific to the `Block` type; it can be applied to any variable or parameter in Rust.

Reply:
Yes, please call it `_block`.